texttospeech/setup.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google LLC
```

Comment:
This needs to use our new setup.py format.

Reply:
Done. Could you file me a bug to update that in toolkit?